source/unified-test-format/tests/valid-pass/poc-transactions-mongos-pin-auto.yml

```diff
@@ -0,0 +1,169 @@
+description: "poc-transactions-mongos-pin-auto"
```

Comment:
Are you aware that the `mongos-pin-auto.yml` test file is auto-generated by [mongos-pin-auto-tests.py](https://github.com/mongodb/specifications/blob/master/source/transactions/tests/mongos-pin-auto-tests.py)?

Reply:
I'm aware, but in this case I was just concerned with porting over two tests that covered the test runner operations.